tests/phpunit/CRM/Membershipextrasimporterapi/EntityCreator/MembershipPaymentTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+use CRM_MembershipExtras_Test_Fabricator_Contact as ContactFabricator;
+use CRM_MembershipExtras_Test_Fabricator_MembershipType as MembershipTypeFabricator;
+use CRM_MembershipExtras_Test_Fabricator_Membership as MembershipFabricator;
+use CRM_MembershipExtras_Test_Fabricator_Contribution as ContributionFabricator;
+use CRM_Membershipextrasimporterapi_EntityCreator_MembershipPayment as MembershipPaymentCreator;
+
+/**
+ *
+ * @group headless
+ */
+class CRM_Membershipextrasimporterapi_EntityCreator_MembershipPaymentTest extends BaseHeadlessTest {
+
+  private $contactId;
+
+  private $membershipId;
+
+  private $contributionId;
+
+  public function setUp() {
+    $this->contactId = ContactFabricator::fabricate()['id'];
+
+    MembershipTypeFabricator::fabricate(['name' => 'Student', 'minimum_fee' => 50]);
+    $this->membershipId = MembershipFabricator::fabricate([
+      'contact_id' => $this->contactId,
+      'membership_type_id' => 'Student',
+      'join_date' => date('Y-m-d', strtotime('-1 year')),
+      'start_date' => date('Y-m-d', strtotime('-1 year')),
+    ])['id'];
+
+    $this->contributionId = ContributionFabricator::fabricate([
+      'contact_id' => $this->contactId,
+      'financial_type_id' => 1,
```

Comment:
We should check for `financial type` and create it if any because it is not reserved

Reply:
I am going to change it to "Member Dues" since API accept names, and given that the test database already comes with 4 default financial types, don't want to overcomplicate things in tests if possible.